client/application.html

```diff
@@ -76,14 +83,23 @@ <h1 class="center">Apply: {{branch}}</h1>
 				{{/each}}
 				{{{endText}}}
 				<div class="center">
+					{{#unless unauthenticated}}
 					{{#if user.applied}}
 						<input type="submit" value="Update" />
 						<button id="delete" style="background-color: #FF4136;">Delete</button>
 					{{else}}
 						<input type="submit" value="Submit" />
 					{{/if}}
+					{{else}}
+						<input type="submit" value="Submit" />
+					{{/unless}}
+
 				</div>
 			</form>
 		{{/sidebar}}
+		<script type="text/javascript">
+		formTypeString = "Application";
```

Comment:
WTF is this a global variable?? Remove the type attribute from the `<script>` tag. If you _need_ a global variable, do `window.myVariable = x` instead so that it will work in strict mode.

Reply:
adding it to `window` won't work because typescript doesn't recognize additional properties on `window`